oauth.js

```diff
@@ -0,0 +1,12 @@
+// This is where we would put environment ID's, just in case we wanted to sign
+// up with google or something.
+
+// var ids = {
+//   facebook: {
+//     clientID: process.env.FB_clientID,
+//     clientSecret: process.env.FB_clientSecret,
+//     callbackURL: process.env.FB_callbackURL
+//   }
+// };
+
+// module.exports = ids;
```

Comment:
idk how environment variables work for heroku, but would it make sense to put all sensitive data (including the mono_uri) into this one file? also, even though nothing is in this, we probably shouldn't upload it because of possible mistakes in the future...